vignettes/corridor-segmentation.Rmd

```diff
@@ -0,0 +1,28 @@
+---
+title: "3. Corridor segmentation"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{3. Corridor segmentation}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(CRiSp)
+```
+
+For a more detailed analysis of an urban river corridor, corridor-level delineation may not be sufficient. The corridor needs to be subdivided into smaller morphological units. Segmentation is a process of subdividing the corridor by using major transversal road or rail infrastructure lines.
+
+By default, the all-in-one function `delineate_corridor()` divides the corridor into segments. It is also possible to use the `delinate_segment()` function to divide the corridor in a separate step. To demonstrate this as a separate step, we will use the built-in `bucharest_corridor` and `bucharest_streets` as input.
+
+```{r eval=FALSE}
+segmented_corridor <- delineate_segments(bucharest_corridor, bucharest_streets)
```

Comment:
the function called here (`delineate_segments()`) is not the same as mentioned in the text above (`delinate_segment()`)

Reply:
Thanks for pointing this out. I updated the text above.